schemas/1.0/general/applicant.json

```diff
@@ -15,7 +15,11 @@
     "home_address": { "$ref": "#/definitions/address" },
     "correspondence_address": { "$ref": "#/definitions/address" },
     "residence_type": {"type": ["string", "null"], "enum": ["rented", "temporary", "parents", "applicant_owned", "partner_owned", "joint_owned", "someone_else", "none", null] },
-    "relationship_to_owner_of_usual_home_address": { "type": ["string", "null"] }
+    "relationship_to_owner_of_usual_home_address": { "type": ["string", "null"] },
+    "has_partner":{ "type":"string", "enum":["yes", "no"] },
```

Comment:
Wondering if this will actually be null for Non means/u18/PSE/Appeal no change/CIFC etc (esp once we move when the question is asked) and we should account for that preemptively